api/config.js

```diff
@@ -37,8 +39,14 @@ config.superuser.email = process.env.HACKILLINOIS_SUPERUSER_EMAIL;
 config.superuser.password = process.env.HACKILLINOIS_SUPERUSER_PASSWORD;
 
 config.auth.secret = config.secret;
-config.auth.header = 'Authorization';
+config.auth.headers = {
+  api: 'API-Authorization',
```

Comment:
This is a bit strange. The [convention](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization) is for the authorization header to go `Authorization: "<type> <value>"`. I think having a separate header for admin override is reasonable, but the actual auth header should not be special.